go/saltpack/basic/key.go

```diff
@@ -235,6 +237,15 @@ func (k SigningPublicKey) ToKID() []byte {
 	return k[:]
 }
 
+// LookupSigningPublicKey allows a signing public key to be used in place of a
+// (public) keyring.
```

Comment:
Can you use the Keyring we made in `libkb/saltpack_verify`? If needs be you can export that out of `libkb`...